### ISSUE TITLE
setup failing to install ansible using ubuntu-setup/ansible.sh

### ISSUE THREAD

**Issue by csantanapr** [openwhisk/openwhisk#530]
Affects vagrant and stand-alone ubuntu
```
Reading state information... Done
E: Version '2.0.2.0-1ppa~trusty' for 'ansible' was not found
sudo: pip: command not found
The program 'ansible' is currently not installed. To run 'ansible' please ask your administrator to install the package 'ansible'
The program 'ansible-playbook' is currently not installed. To run 'ansible-playbook' please ask your administrator to install the package 'ansible'
```

**Comment by csantanapr** [comment]
Root cause is that we specify to strict version of ansible `2.0.2.0-1ppa~trusty`
this version is not longer available, 
It should be more relax the version like `2.*`

**Comment by domdom82** [comment]
actually it needs to be at least 2.0.2.0 I wonder why they keep throwing out previous versions. makes version pinning impossible.

**Comment by csantanapr** [comment]
Using 2.* installs 2.1.0.0 which it works for us

See [here](https://launchpad.net/~ansible/+archive/ubuntu/ansible)

**Comment by domdom82** [comment]
I guess we will be safe. We had problems with 2.0.1.0 that were fixed in 2.0.2.0. So as long as they don't screw up and put up 2.0.1.0 again using "2.*" will work ok. I wonder why they don't just keep old versions for some time longer.
